.github/workflows/aws_deploy.yml

```diff
@@ -75,17 +82,30 @@ jobs:
 
       - name: Get timestamp
         id: timestamp
-        run: echo "timestamp=$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
+        run: echo "timestamp=$(date +%Y%m%d%H%M%S)" >> $GITHUB_OUTPUT
```

Comment:
AFAICT, using `GITHUB_OUTPUT` is the best way to store variables for use in the action.